recipes/mTEDx/ASR/CTC/README.md

```diff
@@ -0,0 +1,47 @@
+# mTEDx ASR with CTC models.
+This folder contains the scripts to train a wav2vec based system using mTEDx. You can train either a single-language wav2vec model or multilingual
+wav2vec model. Before running this recipe, make sure to read this [README](../../README.md) file first.
+
+**Note:**\
+Wav2vec model used in this recipe is pre-trained on the French language.
+In order to use another language, don't forget to change the `wav2vec2_hub`
+in the `train_wav2vec.yaml` YAML file.
+
+
+# How to run
+
+To train a single-language wav2vec model, run:
+```bash
+$ python train.py hparams/train_wav2vec.yaml
+```
+
+To train a multilingual wav2vec model, run:
+```bash
+$ python train.py hparams/train_xlsr.yaml
+```
+
+# Results
+
+TODO
```

Comment:
ok, so we wait with the merge ;-)
Are results there yet - computations pending?

Reply:
added them to the README file